lib/CL/devices/devices.c

```diff
@@ -206,7 +206,7 @@ pocl_init_devices()
      infinite loop. */
 
   if (init_in_progress)
-      return;
+      return CL_SUCCESS; /* debatable, but what else can we do ? */
```

Comment:
I know. My question is which would be the most correct return value in this case. I went with CL_SUCCESS, but maybe something else would be more apt.